poetrytools/poetics.py

```diff
@@ -94,6 +94,21 @@ def num_vowels(syllables):
     return len([syl for syl in syllables if any(char.isdigit() for char in syl)])
 
 
+def get_vowel_index(syllables, n):
```

Comment:
I would really like for this to support negative indexing, so I can get the last syllable with something like `poetrytools.get_vowel_index(foo, -1)`

This does what I want:

```
estab = poetrytools.get_syllables('establishment')[0]

def get_syllable_index(sylls, n):
    indices = [idx for idx, phone in enumerate(sylls) if any(char.isdigit() for char in phone)]
    return indices[n]

print(sylls[get_syllable_index(estab, -1):])
print(sylls[get_syllable_index(estab, 2):])

# ['AH0', 'N', 'T']
# ['IH0', 'SH', 'M', 'AH0', 'N', 'T']
```

Does this do what you want?